### PR TITLE
Fix github url strings (org edx -> openedx)

### PR DESCRIPTION
**Pull request by sarina**
## This PR was autogenerated

This pr replaces the old GitHub organization, github.com/edx, with the new GitHub organization, github.com/openedx.

Tagging @openedx/tcril-engineering for review, but others are welcome to provide review.

Ref: https://github.com/openedx/tcril-engineering/issues/42

### CODE REVIEW

**Comment by justinhynes** [comment]
I'll be taking a look at this today! (Sorry for the delay)

File: pylintrc [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # ** DO NOT EDIT THIS FILE **
 # ***************************
 #
-# This file was generated by edx-lint: https://github.com/edx/edx-lint
+# This file was generated by edx-lint: https://github.com/openedx/edx-lint
```
Comment (justinhynes) [inline pylintrc:5]: @sarina Do you happen to know if this file was updated by the script (modifying the `pylintrc` file directly)? Or was it updated using the `write` command of `edx-lint`?

Trying to do my best to heed the warnings of `** DO NOT EDIT THIS FILE **` (and worried that the update may get overwritten the next time we need to regenerate the file).

Comment (sarina) [reply pylintrc:5]: I did this via edx-lint which creates a special commit message you can see on this PR: 

https://github.com/openedx/credentials/pull/1746/commits/8686a1ab52665f834814bef935f9abf824ac3bbd

Comment (justinhynes) [reply pylintrc:5]: Thanks for the confirmation!

**Review by justinhynes — APPROVED** [review]
LGTM

**Pull request merged**